src/component/forcefield/HyperReducedTetrahedronFEMForceField.h

```diff
@@ -161,7 +161,7 @@ class HyperReducedTetrahedronFEMForceField : public virtual TetrahedronFEMForceF
     using TetrahedronFEMForceField<DataTypes>::_gatherPt; //use in GPU version
     using TetrahedronFEMForceField<DataTypes>::_gatherBsize; //use in GPU version
     using TetrahedronFEMForceField<DataTypes>::drawHeterogeneousTetra;
-    using TetrahedronFEMForceField<DataTypes>::drawAsEdges;
+//    using TetrahedronFEMForceField<DataTypes>::drawAsEdges;
```

Comment:
This is related to: 
https://github.com/sofa-framework/sofa/commit/b2582f1600bf4d945d839cb6612da40f7d5a24a8

Reply:
Ok so the idea is that drawAsEdges was redundant with the wireFrame box in the GUI of SOFA?